transformations/gender_neopronouns/README.md

```diff
@@ -0,0 +1,83 @@
+referring to a person. Some examples include: xe/xem/xyr, ze/hir/hirs, and ey/em/eir. Neopronouns can be used by
+anyone, though most often they are used by transgender, non-binary, and/or gender nonconforming people._
+Source: https://intercultural.uncg.edu/wp-content/uploads/Neopronouns-Explained-UNCG-Intercultural-Engagement.pdf
+
+In the image below, we can find the temporal variations in neopronoun usage over the past few years from  the **_Gender Census 2021: Worldwide Report_**.
+
+<img title=" The neopronoun preferences over times" src="https://gendercensus.files.wordpress.com/2021/03/gc2021-pronouns-over-time-minus-he-she-they.png" width="600" height="354" />
+
+## What type of a transformation is this?
+This transformation performs **_grammatically correct substitution_** of the gendered pronouns he/she in a given sentence with their neopronoun counterparts. The reason why a simple look-up-table approach might not work as depending on the context, the _case_ can belong to the _Nominative/Subject Pronouns,	Accusative/Object Pronouns,	Pronominal Possessive/Possessive Adjectives,	Predicative Possessive/Possessive Pronouns	or Reflexive Pronouns_ categories.
+
+
+Example:
+```
+He like himself. ↔️ Ey likes emself **OR** Ze likes zirself etc.
+```
+The neopronoun list is sourced from [Neopronouns-Explained resource](https://intercultural.uncg.edu/wp-content/uploads/Neopronouns-Explained-UNCG-Intercultural-Engagement.pdf) curated by UNC Greensboro and [LGBTA WIKI](https://lgbta.wikia.org/wiki/Neopronouns).
+Refer to [this](https://github.com/vinayprabhu/neo_pronouns_gen) Github repo for the [dataset construction details](https://github.com/vinayprabhu/neo_pronouns_gen/blob/main/dataset_generate_neopronouns.ipynb), [examples](https://github.com/vinayprabhu/neo_pronouns_gen/blob/main/data/df_examples_neo.csv) and [pronounciation key](https://github.com/vinayprabhu/neo_pronouns_gen/blob/main/data/df_pronounce_neo.csv).
+
+## What tasks does it intend to benefit?
+As things stand, NLP models such as those deployed for neural translation fail to recognize the neopronouns and often treats them as proper nouns. For example, Google translate translates the sentence _Ve likes verself_ to ವೀ ತನ್ನನ್ನು ತಾನೇ ಇಷ್ಟಪಡುತ್ತಾನೆ into Kannada which reads _(Someone named) Vee likes himself_! Also, as revealed in this survey titled [Toward Gender-Inclusive Coreference Resolution](https://arxiv.org/pdf/1910.13913.pdf), _"Only 7.1% (one paper!) considers neopronouns and/or specific singular THEY."_
+This transformation seeks to render the training data used in NLP pipelines more neopronoun aware so as to help tackle the downstream risks of [trans erasure](https://allthingslinguistic.com/post/118373603278/i-am-at-the-end-of-my-first-semester-of).
+
+Benchmark Results:
+* Seq2Seq Generation task ran on IMDB dataset/Roberta model with a 1% sample size. The original accuracy was 96% and the perterbed accuracy was 95%
```

Comment:
nit: perterbed-> perturbed

Reply:
Also, I am unsure what you mean by Seq2Seq model on IMDB with Roberta. This is a classification task and a masked language model, both of which are not seq2seq.